storage/mysql/map_storage_test.go

```diff
@@ -143,43 +144,69 @@ func TestMapRootUpdate(t *testing.T) {
 	s := NewMapStorage(DB)
 
 	ctx := context.Background()
-	tx := beginMapTx(ctx, s, mapID, t)
```

Comment:
Delete the comment at line 141? Or, if the intent of the test was to ensure we can't override a newer root with an older one, then we should add such a test to the table. (The comment seems to suggest that.)